moneymarket.rb

```diff
@@ -0,0 +1,60 @@
+module Bank
+  class MoneyMarketAccount < SavingsAccount
+    MAX_TRANSACTIONS = 6
+    def initialize(id, initial_balance, open_date)
+      super(id, initial_balance, open_date)
+      @current_transactions = 0
+      @transaction_ban = false
```

Comment:
I like that you use this boolean to handle the ability to execute a transaction or not